apiExamples/basic.html

```diff
@@ -0,0 +1,7 @@
+<auro-checkbox-group>
+  <span slot="legend">Form label goes here</span>
+  <auro-checkbox value="checkbox option" name="example1">Checkbox option</auro-checkbox>
+  <auro-checkbox value="checkbox option" name="example2" checked>Checkbox option</auro-checkbox>
+  <auro-checkbox value="checkbox option" name="example3">Checkbox option</auro-checkbox>
+  <auro-checkbox value="checkbox option" name="example4">Checkbox option</auro-checkbox>
+</auro-checkbox-group>
```

Comment:
None of these examples have an ID on the `auro-checkbox` node. None of these examples pass accessibility tests. 

<img width="837" alt="Screen Shot 2023-07-01 at 8 19 34 PM" src="https://github.com/AlaskaAirlines/auro-checkbox/assets/181089/d7de3600-92d8-468f-b49b-efa73ca6f299">

Reply:
What is this tool? I would like to test the accessibility myself as well.

Reply:
@blackfalcon what is the a11y guideline for ID attributes on checkboxes?